### PR TITLE
Update components.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/edb_core/components.py [hunk 1]
```diff
@@ -48,7 +47,7 @@ def resistor_value_parser(RValue):
 
 
 class Components(object):
-    """HFSS 3D Layout class."""
+    """Commponents class."""
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:50]:
```suggestion
    """Components class."""
```

File: pyaedt/edb_core/components.py [hunk 2]
```diff
@@ -892,7 +883,7 @@ def get_pin_position(self, pin):
 
     @aedt_exception_handler
     def get_pins_name_from_net(self, pin_list, net_name):
-        """List pins belonging to a net.
+        """Retrive pins belonging to a net.
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:886]:
```suggestion
        """Retrieve pins belonging to a net.
```

File: pyaedt/edb_core/components.py [hunk 3]
```diff
@@ -356,10 +350,11 @@ def get_component_by_name(self, name):
 
         Returns
         -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
         self.edb.Cell.Hierarchy.Component
             Edb component (Edb.Cell.Hierarchy.Component) object if exists
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:357]:
```suggestion
           Edb component (Edb.Cell.Hierarchy.Component) object if exists
```

File: pyaedt/edb_core/components.py [hunk 4]
```diff
@@ -356,10 +350,11 @@ def get_component_by_name(self, name):
...
         Returns
         -------
-        self.edb.Cell.Hierarchy.Component
-            Edb component (Edb.Cell.Hierarchy.Component) object if exists
-
+        bool
+            ``True`` when successful, ``False`` when failed.
         """
+        self.edb.Cell.Hierarchy.Component
+        Edb component (Edb.Cell.Hierarchy.Component) object if exists
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:357]:
```suggestion
            Edb component (Edb.Cell.Hierarchy.Component) object if exists
```

File: pyaedt/edb_core/components.py [hunk 5]
```diff
@@ -305,12 +297,14 @@ def Others(self):
 
     @property
     def components_by_partname(self):
-        """Return a dictionary by part name.
+        """Return components by part name.
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:302]:
```suggestion
```

File: pyaedt/edb_core/components.py [hunk 6]
```diff
@@ -305,12 +297,14 @@ def Others(self):
...
         
 
         Returns
         -------
         dict
+            Dictionary of components by part name.
+
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:307]:
```suggestion

```

File: pyaedt/edb_core/components.py [hunk 7]
```diff
@@ -330,7 +324,7 @@ def components_by_partname(self):
 
     @aedt_exception_handler
     def get_component_list(self):
-        """Return a list of components EDB objects.
+        """Retrieve a list of a component;s EDB objects.
```
Comment (PipKat) [inline pyaedt/edb_core/components.py:327]:
```suggestion
        """Retrieve a list of a component's EDB objects.
```

File: pyaedt/edb_core/components.py [hunk 8]
```diff
@@ -356,10 +348,11 @@ def get_component_by_name(self, name):
 
         Returns
         -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
         self.edb.Cell.Hierarchy.Component
             Edb component (Edb.Cell.Hierarchy.Component) object if exists
```
Comment (akaszynski) [inline pyaedt/edb_core/components.py:355]:
```suggestion
```

File: pyaedt/edb_core/components.py [hunk 9]
```diff
@@ -441,10 +433,10 @@ def set_component_model(self, componentname, model_type="Spice", modelpath=None,
         componentname : str
             Name of the component.
         model_type : str, optional
-            Type of the model. Choices are ``"Spice"`` and `"Touchstone. "`The default is ``"Spice"``
+            Type of the model. Choices are ``"Spice"`` and `"Touchstone"``. "`The default is ``"Spice"``.
```
Comment (akaszynski) [inline pyaedt/edb_core/components.py:434]:
```suggestion
            Type of the model. Choices are ``"Spice"`` and ``"Touchstone"``. "`The default is ``"Spice"``.
```

File: pyaedt/edb_core/components.py [hunk 10]
```diff
@@ -729,14 +719,14 @@ def update_rlc_from_bom(self, bom_file, delimiter=";", valuefield="Func des", co
...
+            Full path to the BOM file, which is a delimited text file. 
+            Header values needed inside the BOM reader must
             be explicitly set if different from the defaults.
         delimiter : str, optional
             Value to use for the delimiter. The default is ``";"``.
         valuefield : str, optional
             Field header containing the value of the component. The default is ``"Func des"``.
-            ``valuefield`` must contain the value of the component at beginning of the value
+            The value for this parameter must being with the value of the component 
              followed by a space and then the rest of the value. For example, ``"22pF"``.
```
Comment (akaszynski) [inline pyaedt/edb_core/components.py:728]:
```suggestion
            followed by a space and then the rest of the value. For example, ``"22pF"``.
```

**Review by akaszynski — APPROVED** [review]
Tentatively approved, but please have those methods with the @Property decorator changed to nouns rather than verbs.

**Pull request merged**